test/constants.go

```diff
@@ -157,6 +157,7 @@ var (
 	DigitalTwin2       = "gid:AAAAFb3lFaaArUVYqk9VdY1Ct1Y"
 	DigitalTwin3       = "gid:AAAAHNdVLTx1-ExZnjv7nVyRiQc"
 	DigitalTwin4       = "gid:AAAAHC_gEo51KUkbvMdGenKHc6Q"
+	DigitalTwin5       = "gid:AAAAHKOnVUXFSEe5gSqzl0fOHoU"
```

Comment:
nitpick and unrelated to this PR, but thought I'd mention it: can you remove the constants that get decoded to `DOCUMENT_TYPE_DIGITAL_TWIN`? They're not usable anymore. And rename the constants to `Node1...5` or something else that isn't "Digital Twin"

Reply:
removed (DigitalTwin still in kbac though)